packages/EasyErrorHandler/src/Bridge/BridgeConstantsInterface.php

```diff
@@ -6,6 +6,11 @@
 
 interface BridgeConstantsInterface
 {
+    /**
+     * @var string
+     */
+    public const TRANSLATION_NAMESPACE = 'easy-error-handler';
```

Comment:
Const MUST be alpha ordered so this one should be at the very bottom of the file.